conda/recipes/numba-cuda/meta.yaml

```diff
@@ -35,8 +35,10 @@ about:
   home: {{ project_urls["Homepage"] }}
   dev_url: {{ project_urls["Repository"] }}
   doc_url: {{ project_urls["Documentation"] }}
-  license: {{ project_data["license"]["text"] }}
+  license: {{ project_data["license"] }}
   license_family: BSD
-  license_file: LICENSE
+  license_file:
+    - LICENSE
+    - LICENSE.numba
```

Comment:
@isVoid see here

Reply:
Maybe I wasn't looking at the latest source tree. thx!